web/src/main/java/org/akaza/openclinica/controller/openrosa/QueryServiceImpl.java

```diff
@@ -76,7 +76,9 @@ public void process(QueryServiceHelperBean helperBean, SubmissionContainer conta
         helperBean.setItemOrdinal(itemOrdinal);
         helperBean.setItemNode(itemNode);
         helperBean.setItemData(getItemData(helperBean));
-        // helperBean.setResStatus(resolutionStatusDao.findByResolutionStatusId(1));
+        if (helperBean.getItemData() == null) {
```

Comment:
This can be more intuitive by doing something like this:
ItemData id = getItemData(helperBean);
if (id == null)
           helperBean.setItemData(createBlankItemData(helperBean));
else
      helperBean.setItemData(id);